src/legacy/core_plugins/vis_type_timeseries/public/components/vis_editor.js

```diff
@@ -82,6 +81,12 @@ export class VisEditor extends Component {
   updateVisState = debounce(() => {
     this.props.vis.params = this.state.model;
     this.props.vis.updateState();
+    if (this.props.appState) {
+      // This should actually never happen, since this method should only be called when we're in editor
```

Comment:
now when I look at this comment, it might be a bit vague as it says that the body of the condition will never execute. Maybe it's better to add a comment on the line above, something like 'The check might be redundant since this method should only be called when we're in editor'. WDYT? 🤔